### ISSUE TITLE
Office UI Fabric ReactJS Dropdown Accessibility issue

### ISSUE THREAD

**Issue by tpanpalia** [OfficeDev/office-ui-fabric-react#7305]
I am using Office UI Fabric Dropdown in ReactJs. I am testing the application for accessibility issues using Keros for Web.

It is showing this issue for dropdowns:
```
aria-valid-attr-value: ARIA attributes must conform to valid values
Fix the following: Invalid ARIA attribute value: aria-labelledby="usercountryRegion-label" 
Snippet: <div data-is-focusable="true" id="usercountryRegion" tabindex="0" aria-expanded="false" role="listbox" aria-labelledby="usercountryRegion-label" aria-describedby="usercountryRegion-option" aria-activedescendant="usercountryRegion-option" name="country" class="ms-Dropdown root_aecb6435">
```
I searched for some answer and all I got is "Ensure the value inside each attribute is spelled correctly and corresponds to a valid value.", but I could not find a solution to this.

**Comment by KevinTCoughlin** [comment]
Thanks for the report @tpanpalia 

It looks like the fix is to find the missing `#` and  validate that the DOM node exists to be referenced. Nice catch if valid, I haven't confirmed.

As per the ARIA spec:

>The purpose of this technique is to provide a short description for an element that can be read by assistive technologies (AT) by using the aria-labelledby attribute. The aria-labelledby attribute associates an element with text that is visible elsewhere on the page by using an ID reference value that matches the ID attribute of the labeling element. Assistive technology such as screen readers use the text of the element identified by the value of the aria-labelledby attribute as the text alternative for the element with the attribute.
>https://www.w3.org/TR/WCAG20-TECHS/ARIA7.html#ARIA7-examples

**Comment by kkjeer** [comment]
It looks like the dropdown is getting assigned an `aria-labelledby` attribute even when there is no label. @tpanpalia which version of Fabric are you using? #6555 (included in version 6.78.0) ideally should have fixed this by changing [line 229](https://github.com/OfficeDev/office-ui-fabric-react/blob/09f1011e7355358023bb8ec712defbe62f786475/packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx#L229). Also, out of curiosity, are you passing a `label` property to the Dropdown component? (Dropdown should work just fine without a label property, but it might be incorrectly assigning aria-labelledby here).

There is another place in Dropdown where a div has aria-labelledby when there is no label (see [line 464](https://github.com/OfficeDev/office-ui-fabric-react/blob/09f1011e7355358023bb8ec712defbe62f786475/packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx#L464) - I can send a PR to fix that), but based on the Keros snippet @tpanpalia posted, that doesn't seem like it's the issue here.

**Comment by tpanpalia** [comment]
Using Fabric version `6.39.2`. No, I am not passing `label` property to the Dropdown component. But it does have `Label `component with it.

**Comment by kkjeer** [comment]
Thanks for the information @tpanpalia! Could you try updating to Fabric 6.78.0 or later? Alternatively, passing a `label` property to the Dropdown component should prevent the Keros issue.

The reason that the Dropdown does not have a valid `aria-labelledby` attribute is that it is looking for the Label component with an id of "usercountryRegion-label". If the Dropdown has a `label` property, then the Dropdown will create a Label component with that id. In Fabric >= 6.78.0, if the Dropdown does not have a `label` property, it will not have an `aria-labelledby` attribute.
